tests/src/CLRTest.Execute.Batch.targets

```diff
@@ -79,16 +79,23 @@ IF NOT "%COMPlus_GCStress%"=="" (
       <BatchCLRTestEnvironmentCompatibilityCheck Condition="'$(JitOptimizationSensitive)' == 'true'">
     <![CDATA[
 $(BatchCLRTestEnvironmentCompatibilityCheck)
-IF "%COMPlus_JitStress%"=="" IF "%COMPlus_JitStressRegs%"=="" IF "%COMPlus_JITMinOpts%"=="" IF "%COMPlus_TailcallStress%"=="" IF "%COMPlus_TieredCompilation%"=="" goto :Compatible1
+REM JitOptimizationSensitive includes our set of tests which cannot run under:
+REM     JitMinOpts=1
+REM     JitStress=*
+REM     JitStressRegs=*
+REM     TailcallStress=1
+REM
+REM TieredCompilation will use minopts. Therefore it is also included in this
+REM set. Unlike the rest, TieredCompilation=0 should run the JitOptimizationSensitive
+REM tests. The following cannot run the test.
+REN
+REM     TieredCompilation=1
+REM     TieredCompilation=
+IF "%COMPlus_JitStress%"=="" IF "%COMPlus_JitStressRegs%"=="" IF "%COMPlus_JITMinOpts%"=="" IF "%COMPlus_TailcallStress%"=="" IF "%COMPlus_TieredCompilation%"=="0" goto :Compatible
```

Comment:
I'd prefer to make the flow here as consistent as possible with the Bash version, i.e. either break TieredCompilation clause into its own condition here or make that file more similar to this.